tests/baselines/reference/api/tsserverlibrary.d.ts

```diff
@@ -0,0 +1,7692 @@
+/*! *****************************************************************************
```

Comment:
Why is this file here now?

Reply:
Got inadvertently picked up during a branch switch; will remove